packages/fxa-content-server/app/scripts/views/sign_in_password.js

```diff
@@ -89,13 +90,14 @@ const SignInPasswordView = FormView.extend({
     return FormView.prototype.logView.call(this);
   },
 
-  setInitialContext(context) {
+  async setInitialContext(context) {
```

Comment:
Does this need to be async? I don't see any awaits in the function body, and Nimbus.experiments is no longer a promise.

Reply:
You're right. We don't need it anymore. Thanks!